### PR TITLE
Add subchannel blockage test Refs #293

### PR DESCRIPTION
**Pull request by kyriv1980**
This Pr resolves issue #293 

THORS bundle 5B has the same fuel configuration as bundle 2B, except that 0.0711-cm-diam wire-wrap spacers are used to separate the peripheral pins from the duct wall. The half-size spacers are used to reduce the flow in the peripheral flow channels and to cause a flatter radial temperature profile across the bundle. It also means that the flat-to-flat distance is reduced appropiately. The pins have a heated length of  45.7cm. A 0.3175-cm-thick stainless steel blockage plate is located 
10.2cm above the start of the heated zone to block 14 edge and internal channels along the duct wall. 

further information for these validation tests can be found at : https://subchannel-dev.hpc.inl.gov/site/v&v/thors.html

### CODE REVIEW

File: sfr/subchannel/THORS/FFM-5Bdetailed.i
Comment (GiudGiud) [inline sfr/subchannel/THORS/FFM-5Bdetailed.i:1]: You call this detailed but it s more like a visualization file?

Does it really increase the resolution of the output? How?

File: sfr/subchannel/THORS/FFM-5B.i [hunk 1]
```diff
@@ -0,0 +1,255 @@
+# J. T. Han 1977,
```
Comment (GiudGiud) [inline sfr/subchannel/THORS/FFM-5B.i:1]: This isn't the author of the model so let s have some text before that

**Review by GiudGiud — COMMENTED** [review]
Need a documentation page

File: sfr/subchannel/THORS/FFM-5Bdetailed.i
Comment (kyriv1980) [reply sfr/subchannel/THORS/FFM-5Bdetailed.i:1]: It doesn't increase the resolution , it provides better visualization of the subchannel assembly.

File: sfr/subchannel/THORS/FFM-5B.i (hunk 1 above)
Comment (kyriv1980) [reply sfr/subchannel/THORS/FFM-5B.i:1]: Will edit the input file accordingly.

**Review by GiudGiud — APPROVED** [review]

**Comment by GiudGiud** [comment]
looks good

**Pull request merged**